libs/testdata/src/main/kotlin/DummyFeatureToggleService.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.etterlatte.funksjonsbrytere
+
+class DummyFeatureToggleService : FeatureToggleService {
+
+    private val overstyrteBrytere = mutableMapOf<FeatureToggle, Boolean>(
+        FellesFeatureToggle.NoOperationToggle to true
+    )
+
+    fun settBryter(bryter: FeatureToggle, verdi: Boolean) = overstyrteBrytere.put(bryter, verdi)
+
+    override fun isEnabled(toggleId: FeatureToggle, defaultValue: Boolean) =
+        overstyrteBrytere.getOrDefault(toggleId, true)
```

Comment:
skulle denne vært

```suggestion
    override fun isEnabled(toggleId: FeatureToggle, defaultValue: Boolean) =
        overstyrteBrytere.getOrDefault(toggleId, defaultValue)
```
?

Reply:
Haha, ja. Litt misvisande den implementasjonen der.